Remove CentOS7 CentOS7-cuda image build

khabiruddin opened the pull request:
CentOS7 is out of life and therefore, we are not making CentOS7 and Centos7-Cuda image build anymore.

CCCP-4561

khabiruddin committed:
remove centos7 and centos7-cuda from the checks

khabiruddin committed:
remove centos7 and centos7-cuda image building from crontab

khabiruddin committed:
some new images in json file

added centos9_stream, alma8 and alma9 in json file

khabiruddin committed:
These file should be here for backup

These files can be used again if we accidentally erase any of
these files from image-builder server.

khappone commented on scripts/crontab:
```diff
@@ -9,9 +9,9 @@ SHELL=/bin/bash
 ######## cPouta production
 #  m  h  dom  mon  dow   command
-   0  0  1,16 *    *     source $HOME/bin/openrc_cpouta_prod_v3.sh; $HOME/diskimage-builder-scripts.cpouta/scripts/image_create_centos7.sh >> $HOME/log/c/centos7.log 2>&1
+#  0  0  1,16 *    *     source $HOME/bin/openrc_cpouta_prod_v3.sh; $HOME/diskimage-builder-scripts.cpouta/scripts/image_create_centos7.sh >> $HOME/log/c/centos7.log 2>&1
```
Instead of commenting these out just like this, you could comment them out and mark them "Free slot"
Also all of the "Reserved for CentOS-*" should me marked free

khabiruddin replied:
Those slots have been marked as "Free slot"

khappone commented on check_image_building/check_image_building_epouta_production.json:
```diff
@@ -0,0 +1,18 @@
+                "ubuntu-20.04",
+                "ubuntu-22.04",
+                "ubuntu-24.04",
+                "centos8_stream",
```
CentOS 8 stream is also deprecated, and in our customer communcation we said we'd remove it, so I think we should remove it at the same time from imagebuidler.

khabiruddin replied:
centos8_stream has been removed.

khabiruddin committed:
centos 8_stream is EOL

As CentOS-8-Stream is EOL, we needed to remove them.

CCCP-4561

khabiruddin committed:
deprecated image build slot marked as free slot

As we are not building deprecated images, thus marking those slots
as free slots.

CCCP-4561

khabiruddin committed:
centos 8_stream removed from build

As CentOS-8_stream is EOL, this is also removed from the build.

CCCP-4561
